src/pgwire/protocol.rs

```diff
@@ -693,7 +693,7 @@ where
             ExecuteResponse::CreatedIndex { existed } => created!(existed, "42710", "index"),
             ExecuteResponse::CreatedSource { existed } => created!(existed, "42710", "source"),
             ExecuteResponse::CreatedSink { existed } => created!(existed, "42710", "sink"),
-            ExecuteResponse::CreatedView => command_complete!("CREATE VIEW"),
+            ExecuteResponse::CreatedView { existed } => created!(existed, "42710", "view"),
```

Comment:
Is this errcode correct?

Reply:
Yup. 42710 stands for "duplicate object". https://www.postgresql.org/docs/12/errcodes-appendix.html